data/apis/amberdata/beacons/amberdata btc_usd.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "AmberData BTC/USD",
```

Comment:
@bbenligiray we did not have BTC/USD so I added this for both Avalanche mainnet and testnet as per our discussion prior